recipe/meta.yaml

```diff
@@ -9,31 +9,31 @@ source:
   # Update an url to https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   # when the package will be available on PyPI.
   url: https://github.com/Lightning-AI/metrics/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: fb58c830b67902acfd0fc5ddee268a64bdc476c41fb571347c6134c261f1723d
+  sha256: 55df8242752fbb6c46d5c54447f09c6991e490ed9580a08b24f82382d3c61048
 
 build:
-  number: 1
+  number: 0
   # pytorch hasn't python 3.11 support on ppc64le: ppc requires gcc v8 (see cbc.yaml). Python 3.11 is incompatible with anything built with gcc<11 because of runtime conflicts.
-  skip: True  # [py<37 or (py>310 and (linux and ppc64le))]
+  skip: True  # [py<38 or (py>310 and (linux and ppc64le))]
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   host:
     - python
-    - numpy
     - pip
     - setuptools
     - wheel
   run:
     - python
-    - numpy >=1.17.2
-    - packaging  # hotfix for utils, can be dropped with lit-utils >=0.5
+    - numpy >=1.20.0
```

Comment:
```suggestion
    - numpy >1.20.0
```

Reply:
https://github.com/Lightning-AI/torchmetrics/blob/520625c3531bf158e1f57b93016a338b10de126e/requirements.txt#L4

Reply:
Indeed, but be careful with requirements files. They are far from the truth. In this case it's right, but they can't necessarily be trusted. `requirements.txt` files (and co) are not directly consumed by any build backends. They are sometimes consumed inside `setup.py`, but when that is the case, all bets are off and you need to look at the code to see how the file is being processed.

https://inspector.pypi.io/project/torchmetrics/1.1.2/packages/fa/86/264f2a30d7cdd2f7e93dcca840b588c002e1631d4dfc1fd6ffa4bb497d23/torchmetrics-1.1.2-py3-none-any.whl/torchmetrics-1.1.2.dist-info/METADATA#line.31

Reply:
thank you for explaining why, 

I find this information very useful, and I believe we should start using `pypi` and the inspector for our estimations.